eventmesh-openconnect/eventmesh-openconnect-java/build.gradle

```diff
@@ -32,4 +29,7 @@ dependencies {
 
     compileOnly 'org.projectlombok:lombok'
     annotationProcessor 'org.projectlombok:lombok'
+
+    testRuntimeOnly "org.apache.logging.log4j:log4j-core"
+    testRuntimeOnly "org.apache.logging.log4j:log4j-slf4j2-impl"
```

Comment:
How about depending on `eventmesh-common` module? To achieve "the logging backends are now only in `eventmesh-common`".